screens/CreateWorkout.js

```diff
@@ -61,6 +61,13 @@ class CreateWorkoutScreen extends React.Component {
   componentDidMount() {
     this.props.navigation.setParams({ save: this.saveWorkout });
     this.props.navigation.setParams({ goHome: this.goHome });
+    firebase.database().ref('/common/workoutCategories/').once('value').then((snapshot) => {
+      let workoutCategories = []
+      snapshot.forEach(function (category) {
+        workoutCategories.push(category.val())
+      });
+      this.setState({ category: workoutCategories[0], workoutCategories });
```

Comment:
Do we need to keep workoutCategories in state or it can be moved just to a property of this class? 
extra render call on setState

Reply:
I tried both of them! When, I keep it in state UI seemed smoother.